globalid.gemspec

```diff
@@ -5,7 +5,7 @@ Gem::Specification.new do |s|
   s.summary     = 'Refer to any model with a URI: gid://app/class/id'
   s.description = 'URIs for your models makes it easy to pass references around.'
 
-  s.required_ruby_version = '>= 2.2.2'
+  s.required_ruby_version = '>= 1.9.3'
```

Comment:
Can we also update the version (line 4) to `0.5.1.doxo`?

Reply:
done